daemon/src/connection.rs

```diff
@@ -611,13 +616,16 @@ pub async fn connect(
                 }
 
                 let num_addresses = maker_addresses.len();
-                let seconds = CONNECT_TO_MAKER_INTERVAL.as_secs();
+
+                // Apply a random number of seconds between the reconnection
+                // attempts so that all takers don't try to reconnect at the same time
+                let seconds = thread_rng().gen_range(5, MAX_RECONNECT_INTERVAL_SECONDS);
```

Comment:
Given that it might take some time to the daemon to reconnect to the maker we could think about showing more information (e.g. the seconds) in the UI.

Reply:
interesting idea! will consider it for the follow-up (for backporting, I'd prefer not to touch the UI)